deeplake/core/vectorstore/vector_search/dataset/dataset.py

```diff
@@ -152,11 +163,15 @@ def create_dataset(
     exec_option,
     embedding_function,
     overwrite,
+    runtime,
     **kwargs,
 ):
-    runtime = None
-    if exec_option == "tensor_db":
-        runtime = {"tensor_db": True}
+    if exec_option == "tensor_db" and (
+        runtime is None or runtime == {"tensor_db": False}
+    ):
+        raise ValueError(
+            "When using `exec_option = 'tensor_db'` you must specify `runtime = {'tensor_db': True}` when creating a new Vector Store."
```

Comment:
@adolkhan We should add an explanation of why this problem happens, not just what parameters need to be changed. Pls change this to:

To execute queries using `exec_option = 'tensor_db'`, the Vector Store must be stored in Deep Lake's Managed Tensor Database. To create the Vector Store in the Managed Tensor Database, specify `runtime = {'tensor_db': True}` when creating the Vector Store.